scipy/spatial/qhull.pyi

```diff
@@ -0,0 +1,98 @@
+'''
+Static type checking stub file for scipy/spatial/qhull.pyx
+'''
+
+from typing import TYPE_CHECKING, List, Tuple, Any
+
+if TYPE_CHECKING:
+    from numpy.typing import ArrayLike
+else:
+    ArrayLike = Any
+
+import numpy as np
+
+class _Qhull:
+    def __init__(self, points: ArrayLike, furthest_site: bool = ...,
```

Comment:
Recommend formatting in the way we discussed in https://github.com/scipy/scipy/pull/13331#pullrequestreview-560737179.

Reply:
Done